### PR TITLE
[AT][ABC][submenu]TestSortinglanguagesByLetter_AfterSelectingLetterInSubmenuI<IrynaDanilevska>

### PR DESCRIPTION
**Pull request by IrynaDanilevska**
[US] - https://trello.com/c/ST9avavI/182-usfunctionalabcsubmenu-search-for-language-by-letter-or-number-name
[TC] - https://trello.com/c/HEg5XCAa/186-tcabcsubmenu-verify-that-languages-are-sorted-and-displayed-on-the-page-abc-by-selected-letter-of-submenuirynadanilevska
[AT] - https://trello.com/c/rHz6o5mM/189-atabcsubmenu-testsortinglanguagesbyletterafterselectingletterinsubmenuhappypathirynadanilevska

### CODE REVIEW

**Comment by NataliiaOliver** [comment]
can you add a link of PR to AT please

File: src/test/java/IrynaDanilevskaTest.java [hunk 1]
```diff
@@ -38,6 +39,44 @@ public void testSearchLanguageField_HappyPath() {
...
+
+        getDriver().get(BASE_URL);
+
+        WebElement browseLanguageMenu = getDriver().findElement(
+                By.xpath("//div[@id='navigation']/ul//a[@href='/abc.html']")
+        );
+        browseLanguageMenu.click();
+
+        WebElement submenuElementLettersNumbers = getDriver().findElement(
+                //By.xpath("//ul[@id='submenu']//a[@href='".concat(LETTER).concat(".html'] "))
```
Comment (VictoriaLema) [inline src/test/java/IrynaDanilevskaTest.java:55]: Удалите, пожалуйста, комментарий

**Comment by IrynaDanilevska** [comment]
[[AT][ABC][submenu] testSortinglanguagesByLetter_AfterSelectingLetterInSubmenu_HappyPath<IrynaDanilevska>](https://trello.com/c/rHz6o5mM/189-atabcsubmenu-testsortinglanguagesbyletterafterselectingletterinsubmenuhappypathirynadanilevska)

File: src/test/java/IrynaDanilevskaTest.java (hunk 1 above)
Comment (IrynaDanilevska) [reply src/test/java/IrynaDanilevskaTest.java]: done

**Review by AramH20 — APPROVED** [review]

**Review by VictoriaLema — APPROVED** [review]

**Review by NataliiaOliver — APPROVED** [review]

**Pull request merged**